src/scenes/ProgressReports/Detail/ProgressReportDrawer/Steps/PromptVariant/PromptVariantStep.tsx

```diff
@@ -16,13 +16,18 @@ interface PromptVariantStepProps {
     { input: UpdatePromptVariantResponse }
   >;
   createItemDocument: DocumentNode<unknown, { input: ChoosePrompt }>;
+
+  formTitle: string;
+  formSubtitle: string;
```

Comment:
I think these are better names that more clearly convey usage:
```suggestion
  title: ReactNode;
  promptInstructions: ReactNode;
```
Labels should always be `ReactNodes` as well to allow html elements as needed.

Whether or not we expect the nodes to provide the typography or not is up for debate but I'd lean towards letting them define it.

Spelled out:
```tsx
title={(
  <Typography variant="h3" gutterBottom>
    Share a story from the community.
  </Typography>
)}
promptInstructions={(
  <Typography variant="body2" gutterBottom>
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Elit pharetra enim justo, molestie amet viverra faucibus. Egestas congue felis 
  </Typography>
)}
```